Source/MQTTnet/Adapter/MqttConnectingFailedException.cs

```diff
@@ -5,12 +5,13 @@ namespace MQTTnet.Adapter
 {
     public class MqttConnectingFailedException : MqttCommunicationException
     {
-        public MqttConnectingFailedException(MqttClientConnectResultCode resultCode)
+        public MqttConnectingFailedException(MqttClientAuthenticateResult resultCode)
             : base($"Connecting with MQTT server failed ({resultCode.ToString()}).")
```

Comment:
Is the message still working when using the new object or does it only print the class name "MqttClientAuthenticateResult"?

Reply:
No, it was supposed to have the correct field referenced.  It apparently got lost when I setup the pull request from my development branch.  Now correctly references the ResultCode field